lib/chromic_pdf/supervisor.ex

```diff
@@ -179,14 +179,16 @@ defmodule ChromicPDF.Supervisor do
                  required(:attribute) => binary()
                }}
 
+      @type navigate_option ::
```

Comment:
I just realized wouldn't they be all plural btw?  `options`

Reply:
Nah, it's read like this: "A single `navigate_option` is either this or that or that" -> Then where these are used in the function specs like

```
@spec print_to_pdf(..., [pdf_option()])
```

ie. `print_to_pdf` receives a list of `pdf_option` thingys